contracts/Anchor.toml

```diff
@@ -1,7 +1,7 @@
 [programs.localnet]
-ocr2 = "My11111111111111111111111111111111111111111"
-access_controller = "My11111111111111111111111111111111111111112"
-deviation_flagging_validator = "My11111111111111111111111111111111111111113"
+ocr2 = "Fh7uhkZLvogdxrQccDpRgMDyN3wKb5ZQGRcpvujeqAae"
```

Comment:
All of these and contract internal ProgramIDs should be reverted back to their dummy value:

- `My11111111111111111111111111111111111111111`
- `My11111111111111111111111111111111111111112`
- `My11111111111111111111111111111111111111113`

cc @archseer how is this exactly envisioned? After the accounts have been set up (could also be part of the script), we'd need a script to find/replace all relevant strings for these ad-hoc tests? Then we recompile and upload the contracts, right?

I think @aalu1418 mentioned recently he would love to have a script like that because there are multiple manual steps currently, and one can easily be skipped.

Reply:
`Anchor.toml` only needs changing if you're using `anchor` or touching the contract tests.

For devnet/testnet/mainnet I intend to conditionally include the ids so they can be selected at build time (`anchor build -- --features testnet`). We could do a similar thing for localnet if we distributed the same keypair amongst ourselves.

Reply:
To automate this, you could use `solana-keygen new -o key.json`, `solana-keygen pubkey key.json` and then `sed` that into the contract as a part of the job steps.

Reply:
Thx @archseer! Added two tasks for this:

- https://github.com/smartcontractkit/chainlink-solana/issues/29
- https://github.com/smartcontractkit/chainlink-solana/issues/30

Reply:
Set placeholders back and added `sed` for now